pkg/devserver/devserver.go

```diff
@@ -94,18 +93,6 @@ func start(ctx context.Context, opts StartOpts) error {
 		redis_state.WithKeyGenerator(redis_state.DefaultKeyFunc{
 			Prefix: "{state}",
 		}),
-		redis_state.WithFunctionCallbacks(func(ctx context.Context, i state.Identifier, rs enums.RunStatus) {
-			switch rs {
-			case enums.RunStatusRunning:
-				// Add this to the in-memory tracker.
-				// XXX: Switch to sqlite.
-				s, err := sm.Load(ctx, i.RunID)
-				if err != nil {
-					return
-				}
-				t.Add(s.Event()["id"].(string), i)
-			}
-		}),
```

Comment:
This breaks some of the GraphQL resolvers at `event.functionRuns`. Not sure if they are still used. They were by TS integration tests, but rewriting.

Reply:
Cool, OK.  This is definitely no longer necessary so a refactor is ideal!